judge/models/comment.py

```diff
@@ -151,3 +151,16 @@ class Meta:
         unique_together = ['voter', 'comment']
         verbose_name = _('comment vote')
         verbose_name_plural = _('comment votes')
+
+class CommentLock(models.Model):
+    page = models.CharField(max_length=30, verbose_name=_('associated Page'), db_index=True,
+                            validators=[RegexValidator('^[pc]:[a-z0-9]+$|^b:\d+$|^s:',
+                                                       _('Page code must be ^[pc]:[a-z0-9]+$|^b:\d+$'))])
```

Comment:
I suppose we might want this for editorial/blog comments too.

Reply:
It does have that though. Notice the ^b and ^s.

Reply:
Oh yikes